public/src/actions/campaign.jsx

```diff
@@ -30,27 +44,15 @@ export function saveNewCampaign(campaignInfo, history) {
   });
 }
 
-export function setCampaignsList(campaignsList) {
-  return {
-    type: SET_CAMPAIGNS,
-    payload: campaignsList
-  };
-}
-
-export function setCurrentCampaign(campaignDataObj) {
-  return {
-    type: SET_CAMPAIGN_CURRENT,
-    payload: campaignDataObj
-  };
-}
 
 export function fetchAllCampaigns() {
   return dispatch => axios.get('/campaigns', {
     headers: { Authorization: ` JWT ${localStorage.getItem('auth_token')}` }
   })
   .then((campaigns) => {
     const { data: campaignsList } = campaigns;
-    return dispatch(setCampaignsList(campaignsList));
+    dispatch(setCampaignsList(campaignsList));
+    return campaigns;
```

Comment:
Is there a reason we're returning campaigns here? I thought we decided this was unnecessary.